src/index.ts

```diff
@@ -1,4 +1,4 @@
-module.exports = {
+export = {
```

Comment:
`export default`?

Reply:
This will compile to `exports['default']`

Reply:
This is old syntax, docs recommend to use `default`

Reply:
Fixed